w3c/aria-at-app
# Fix Test Reports Pages to work without Cycles

### Summary of changes
- Add new reducer for `getPublishedRuns`
- Remove use of Cycle reducers from the the `ResultsPage` and `RunResultsPage` components
- Adjust `getNewTestVersions` service to be a `getTestVersions` service and do the filtering to only show new test versions in the `ConfigureActiveRuns` component

### QA Instructions:
- Ensure you have publshed tests locally (Create active runs, runs some runs, marks some runs as final (you don't need to actually complete the runs))
- Go to "Test Reports" page. Does it work? Do all the links on it work?
